docusaurus/docs/packages/pkg/relayer/relayminer_config.md

```diff
@@ -0,0 +1,379 @@
+required.
+
+## `service_config`
+_`Required`_
+
+The `service_config` section of the supplier configuration is a set of options
+that are specific to the service that the `RelayMiner` will be offering to the
+Pocket network.
+
+### `url`
+_`Required`_
+
+The URL of the service that the `RelayMiner` will forward the requests to when
+a relay is received. It must be a valid URL (not just a host) and must be reachable
+from the `RelayMiner` instance.
+
+### `authentication`
+_`Optional`_
+
+The `authentication` section of the supplier configuration is a pair of `username`
+and `password` that will be used by the basic authentication mechanism to authenticate
+the requests that are forwarded to the service.
+
+### `headers`
+_`Optional`_
+
+The `headers` section of the supplier configuration is a set of key-value pairs
+that will be added to the request headers when the `RelayMiner` forwards the
+requests to the service. It can be used to add additional headers like
+`Authorization: Bearer <TOKEN>` for example.
+
+## `hosts`
+_`Required`_, _`Unique` for each referenced proxy_, _`Unique` within the supplier's `hosts` list_
+
+The `hosts` section of the supplier configuration is a list of hosts that the
+`RelayMiner` will accept requests from. It must be a valid host that reflects
+the on-chain supplier staking service endpoints.
+
+It is used to determine which RPC-Type the request is for and must be unique
+across all the `hosts` lined to a given proxy.
+
+_Note: The `name` of the supplier is automatically added to the `hosts` list_
+
+## `proxy_names`
+_`Required`_, _`Unique` within the `proxy_names` list_
+
+The `proxy_names` section of the supplier configuration is the list of proxies
+that the `RelayMiner` will use to serve the requests for the given supplier entry.
+
+It must be a valid proxy name that is defined in the `proxies` section of the
+configuration file, must be unique across the supplier's `proxy_names`  and the
+`supplier` `type` must match the `type` of the referenced `proxy`.
+
+# Examples
+
+## Proxy -> Supplier referencing
+
+To illustrate how the `proxy_names` and `name` fields are used to reference
+proxies and suppliers, let's consider the following configuration file:
+
+```yaml
+proxies:
+  - name: http-example
+    ...
+  - name: http-example-2
+suppliers:
+  - name: ethereum
+    ...
+    proxy_names:
+      - http-example
+      - http-example-2
+  - name: 7b-llm-model
+    ...
+    proxy_names:
+      - http-example
+```
+
+In this example, the `ethereum` supplier is referencing two proxies, `http-example`
+and `http-example-2` and the `7b-llm-model` supplier is referencing only the
+`http-example` proxy. This would result in the following setup:
+
+```
+- http-example
+  - ethereum
+  - 7b-llm-model
+- http-example-2
+  - ethereum
+```
+
+## Full config example
+
+Below is a full and commented example of a `RelayMiner` configuration file.
+
+```yaml
```

Comment:
Make sure to reflect changes in #284.

Even better, link to `localnet/poktrolld/config/relayminer_config_full_example.yaml` to avoid maintaining two sources of truth.

Even better, I'd look into adding a live codeblock: https://docusaurus.io/docs/api/themes/@docusaurus/theme-live-codeblock

Reply:
The plugin does not allow to reference/embed external pages, so we would still need to copy paste the content into it.